### ISSUE TITLE
audio stays muted even when toggling the audio button on full screen video

### ISSUE THREAD

**Issue by hadjisoteriou** [christianselig/apollo-bugs#1101]
audio stays muted even when toggling the audio button on full screen video (ie. when you press a post to view the full media), expecting to hear it.  

reproducible on ios 14.0.1 on a non-jailbroken iphone 7

**Comment by andrehsu** [comment]
For me videos become permanently muted after you mute then unmute, or after the video restarts after reaching the end.

**Comment by N0rimaki** [comment]
> For me videos become permanently muted after you mute then unmute, or after the video restarts after reaching the end.

second this

watch the video, close it, open it again and it's muted. you can press the unmute button but audio is still not playing.
1.10.4
iphone 11 ios 14.1

**Comment by geromejumps** [comment]
> > For me videos become permanently muted after you mute then unmute, or after the video restarts after reaching the end.
> 
> second this
> 
> watch the video, close it, open it again and it's muted. you can press the unmute button but audio is still not playing.
> 1.10.4
> iphone 11 ios 14.1

Same issue here
1.10.4
iPhone X ios 14.0

**Comment by rafaelmelo2000** [comment]
It always happens that when I open a video I can hear the audio, but if I close it and reopen it again, I don't hear anything, I have to close and reopen the subreddit to hear audio. Here is a video 
iPhone X on 13.5
Jailbroken
Apollo 1.10.4
https://youtu.be/ns3RkRVJJV4

**Comment by exp111** [comment]
You can fix it by going out of the video and pressing the unmute button on the bottom right (not the player one). 

Works for me, but same bug in general.

**Comment by rafaelmelo2000** [comment]
Yes, I knew it works doing that trick, but wanted to report the bug so it can be corrected in a future update.
By the way, great app!

**Comment by mariebks** [comment]
I second this. Noticed it after upgrading to 12 Pro from X.
